.github/workflows/github-template-update-downstream.yml

```diff
@@ -100,7 +100,8 @@ jobs:
           git add .github/ CONTRIBUTING.md SECURITY.md renovate.json
           if (-not (git diff --staged --quiet)) {
               git commit -m "Sync template files from ${{ inputs.templateRepoName }}"
-              git push origin main
+              gh pr create --fill
```

Comment:
I think you need to extract the PR number or url from this command, so you can pass it to the merge command. Also, the repo has multiple upstream, I don't know if the command will infer the repo correctly. You can set it using `--repo` if needed.